index.js

```diff
@@ -123,19 +123,31 @@ window.onclick = (e) => {
     closeModal();
   }
 };
-
-function closeModal() {
-  modal.classList.remove("active");
-  shadow.classList.remove("active");
-  let counter = 0;
+function handleLevel(string)
+{
+    if(string==='Easy')
+timerStart();
+    else if(string==='Medium')
+timerStart();
+    else if(string==='Hard')
+timerStart();
+    
+}
+function timerStart(){

```

Comment:
as I suggested in my previous comment, do not make this a separate function, there is this anonymous function just change it to a named function, and then instead of calling the timerStart(), call that function passing the level to it. The timer logic was already there in that function

Reply:
It's not working when I'm doing this idk why :(

Reply:
Made changes !